segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaConnectionListener.java

```diff
@@ -110,7 +110,7 @@
     @VisibleForTesting
     public PravegaConnectionListener(boolean enableTls, int port, StreamSegmentStore streamSegmentStore, TableStore tableStore, ScheduledExecutorService tokenExpiryExecutor) {
         this(enableTls, false, "localhost", port, streamSegmentStore, tableStore, SegmentStatsRecorder.noOp(), TableSegmentStatsRecorder.noOp(),
-                new PassingTokenVerifier(), null, null, true, tokenExpiryExecutor);
+                new PassingTokenVerifier(), null, null, true, 10 * 1024 * 1024, tokenExpiryExecutor);
```

Comment:
Such as this constructor. That value could be a nice default in a builder, along with all the nulls and no-op classes.